### ISSUE TITLE
Pull requests merging `production` into a branch auto-deploy an invalid instance

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7-test-panel#102]
[This pull request](https://github.com/specify/specify7/pull/3776) caused a massive issue as the test panel would auto-deploy it, assuming that production was a tag on our docker hub. Of course, that was not the case, so it would error if it ever tried to deploy anything.

When attempting to run the compose process:

```
ERROR: manifest for specifyconsortium/specify7-service:production not found: manifest unknown: manifest unknown
```

This prevented any new deployments from being created and introduced continuous problems

**Comment by grantfitzsimmons** [comment]
For reference, here are all of the tags we have on our DockerHub:
https://hub.docker.com/r/specifyconsortium/specify7-service/tags

**Comment by maxxxxxdlp** [comment]
Yeah, `production` is a special branch and is available in docker as `edge` instead.

Need to modify the test server code to substitute that in the response received from github API (when fetching pull requests)
